rpc/send_test.go

```diff
@@ -156,6 +156,31 @@ func TestUnretryableError(t *testing.T) {
 	}
 }
 
+// TestClientNotReady verifies that Send gets an RPC error when a client
+// does not become ready.
+func TestClientNotReady(t *testing.T) {
+	rpcContext := createNewTestRPCContext(t)
+
+	opts := Options{
+		N:               1,
+		Ordering:        OrderStable,
+		SendNextTimeout: 1 * time.Second,
+		Timeout:         1 * time.Second,
```

Comment:
I've just checked the TestComplexScenarios test - also takes about 0.5-1sec. Likely due to all the servers getting started. Doesn't matter too much but it's good to pay some attention to keeping the tests speedy -  having a fast test suite is nice.